index.Rmd

```diff
@@ -695,7 +701,7 @@ class: inverse, center, middle, hide-count
 
 .bg-washed-green.b--dark-green.ba.bw2.br3.shadow-5.ph4.mt5[
 .left[
-_.orange[The best] is the enemy of the good!_ <br> .center[... but ...] <br> .right[_The only way to go fast is to .orange[GO WELL!]_]
+_.orange[The best] is the enemy of the good!_ <br> .center[... and ...] <br> .right[_The only way to go fast is to .orange[GO WELL!]_]
```

Comment:
come mai metti and? io lo vedo come un contrasto, a cui trovare il giusto equilibrio, con and, mi sembrerebbe più una somma di eccessi. Che dici?

Reply:
"and" perché non sono due citazioni in contrasto, una definisce l'obiettivo ( "good enough") mentre l'altra come raggiungerlo ("go well")